recipes/fs/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "fs" %}
+{% set version = "2.0.9" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8570c6349db3b0bf68fe615fe2683ca04796082aceae0ae1d232f71a6f6e47ef" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - appdirs >=1.4.3,<2
+    - pytz
+    - six >=1.10.0,<2
+    - enum34 >=1.1.6,<2  # [py2k]
+    - scandir >=1.5,<2  # [py2k or py34]
```

Comment:
what about py35 and py36

Reply:
fs.egg-info/requires.txt said
[scandir :python_version < '3.5']
scandir~=1.5

seems like scandir is built-in from 3.5 on?  https://docs.python.org/3/whatsnew/3.5.html

Reply:
`scandir` is an optional dependency, since it is a C module and may not be available to non-CPython implementations. I patched the library a while ago to work without it, but it will be faster if `scandir` is available.

Reply:
unfortunately, conda doesn'y yet support optional dependencies.

Reply:
also, conda-forge recipes get built only on three specific platforms, all CPython

Reply:
I guess its fine as long as installing scandir on Py35+ doesn't break anything.

Reply:
@willmcgugan the comment after scandir is a selector that says the line is only relevant on py2k or py34

Reply:
I am currently also looking on this because I am interested in fs.sshfs and this is the base for this.

https://github.com/conda-forge/staged-recipes/pull/4055

How does it go on here?